modules/qna/src/backend/utils.ts

```diff
@@ -23,40 +23,63 @@ export const getQnaEntryPayloads = async (
   defaultLang: string,
   bp: typeof sdk
 ) => {
-  let args: any = {
+  let lang = event.state?.user?.language ?? defaultLang
+  if (!qnaEntry.answers[lang] && !qnaEntry.contentAnswers[lang]) {
```

Comment:
good that you included that but it should never happen since you can't create a qna without no answer